content/artifacts/artifacts-finder/plugin.js

```diff
@@ -168,8 +177,8 @@ class ArtifactsFinder {
       this.logs.appendChild(createDom("div", "Start Finding"));
       this.prospectArtifacts();
       setTimeout(this.findArtifacts.bind(this), 0);
-      this.prospectTimerId = setInterval(this.prospectArtifacts.bind(this), AUTO_INTERVAL);
-      this.findTimerId = setInterval(this.findArtifacts.bind(this), 10000);
+      this.prospectTimerId = setInterval(this.prospectArtifacts.bind(this), LOOP_INTERVAL_MS);
+      this.findTimerId = setInterval(this.findArtifacts.bind(this), LOOP_INTERVAL_MS);
```

Comment:
Do not change this line, find should start asap

Reply:
OK thanks for letting me know. I'll revert it.

Reply:
Added commit to revert the second line above, marking as resolved